packages/javascript/bh-shared-ui/src/views/TierManagement/Save/SelectorForm/SelectorForm.test.tsx

```
@@ -246,24 +256,28 @@ describe('Selector Form', () => {
 
-    test('filling in the name value submits the form and navigates back to the details page', async () => {
+    test('filling in the name value allows updating the selector and navigates back to the details page', async () => {
         const history = createMemoryHistory({
             initialEntries: [detailsPath, editExistingPath],
         });
 
         render(
             <Routes>
                 <Route path={'/'} element={<SelectorForm />} />;
-                <Route path={'/tier-management/edit/tag/:tagId/selector/:selectorId'} element={<SelectorForm />} />;
+                <Route path={'/tier-management/save/tier/:tierId/selector/:selectorId'} element={<SelectorForm />} />;
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)